### PR TITLE
Changes made to pick ppc64le fdinfo flags.

### PR DESCRIPTION
**Pull request by sudeeshjohn**
Changes made to pick ppc64le fdinfo flags. Also the failure reporting method.

Signed-off-by: Sudeesh John <sudeesh@linux.vnet.ibm.com>

### CODE REVIEW

**Comment by sudeeshjohn** [comment]
check on python 2.6 is failing due to below error;

 "   ERROR: You need Python 2.7 or 3.3+ to install the typing package.
    
    ----------------------------------------
    Command "python setup.py egg_info" failed with error code 1 in /tmp/pip-build-xrXup5/typing
The command "pip install Sphinx" failed and exited with 1 during 

"

File: libvirt/tests/src/conf_file/qemu_conf/auto_dump.py [hunk 1]
```diff
@@ -108,13 +107,28 @@ def run(test, params, env):
...
         utils.run('kill %s' % iohelper_pid)
 
-        # Check if bypass cache flag set or unset accordingly.
-        if (flags & 040000) and bypass_cache != '1':
-            raise error.TestFail('auto_dump_bypass_cache is %s but flags '
-                                 'is %o' % (bypass_cache, flags))
-        if not (flags & 040000) and bypass_cache == '1':
-            raise error.TestFail('auto_dump_bypass_cache is %s but flags '
-                                 'is %o' % (bypass_cache, flags))
+        arch = utils.get_arch()
```
Comment (balamuruhans) [inline libvirt/tests/src/conf_file/qemu_conf/auto_dump.py]: we can avoid using using `utils from autotest`, instead python's `platform` module can be used.
```
import platform
arch = platform.machine()
```

File: libvirt/tests/src/conf_file/qemu_conf/auto_dump.py [hunk 2]
```diff
@@ -31,8 +31,8 @@ def run(test, params, env):
     vm = env.get_vm(vm_name)
 
     if panic_model and not libvirt_version.version_compare(1, 3, 1):
-        raise error.TestNAError("panic device model attribute not supported"
-                                " on current libvirt version")
+        raise test.cancel("panic device model attribute not supported"
```
Comment (balamuruhans) [inline libvirt/tests/src/conf_file/qemu_conf/auto_dump.py]: `raise` is not required here and elsewhere for test.cancel(),

Comment (sudeeshjohn) [inline libvirt/tests/src/conf_file/qemu_conf/auto_dump.py]: my bad.. I didnt check that..thanks for pointing it out..

File: libvirt/tests/src/conf_file/qemu_conf/auto_dump.py [hunk 3]
```diff
@@ -31,8 +31,8 @@ def run(test, params, env):
     vm = env.get_vm(vm_name)
 
     if panic_model and not libvirt_version.version_compare(1, 3, 1):
-        raise error.TestNAError("panic device model attribute not supported"
-                                " on current libvirt version")
+        test.cancel("panic device model attribute not supported"
+                          "on current libvirt version")
```
Comment (balamuruhans) [inline libvirt/tests/src/conf_file/qemu_conf/auto_dump.py]: pylint style fails, here and elsewhere
```
test.cancel("panic device model attribute not supported"
            "on current libvirt version")
```

**Pull request merged**